### PR TITLE
IGNITE-11098 Web console: font cleanup

### PR DESCRIPTION
**Pull request by Klaster1**
https://issues.apache.org/jira/browse/IGNITE-11098
@AlexDel please review.

### CODE REVIEW

File: modules/web-console/frontend/app/components/page-configure-advanced/style.scss [hunk 1]
```diff
@@ -94,40 +94,3 @@ page-configure-advanced {
         margin-top: -10px;
     }
 }
-
```
Comment (AlexDel) [inline modules/web-console/frontend/app/components/page-configure-advanced/style.scss]: Is this edition relevant to font changing? Or is it just a side cleanup?

Comment (Klaster1) [reply modules/web-console/frontend/app/components/page-configure-advanced/style.scss]: I found that the .pca-panel class was left unused, so instead of removing "font-family", I removed the whole thing.

Comment (AlexDel) [reply modules/web-console/frontend/app/components/page-configure-advanced/style.scss]: Ok

**Comment by dspavlov** [comment]
The ticket is resolved, closing PR. 
 
Please close PRs for your resolved tickets and other non-needed PRs. 
 
Alternatively, ask a committer 
- to use GitHub UI (committer can associate accounts here: https://gitbox.apache.org/setup/ )
- or /scripts/apply_pull_request.sh from Apache Ignite source.
- or mention in commit message 'Fixes #(a number of PR)'. 
 
These methods will automatically close PR.
 
Feel free to reopen this PR if it is actual.
